database/migrations/000073_sdlc_entities.up.sql

```diff
@@ -0,0 +1,18 @@
+-- Copyright 2024 Stacklok, Inc
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+ALTER TYPE entities ADD VALUE 'release';
```

Comment:
could we add a transaction here so either the migration proceeds or fails as a whole?

Reply:
I remember what happened when we hit this bug so this is good feedback 😃

Reply:
Yeah good call, added it :+1: